### PR TITLE
nit fix: description of access token ins sourcegraph VS Code Extensions in VS Code settings

### PR DESCRIPTION
**Pull request by Siddhant-K-code**
**TL;DR;**

* Fixed Description of `sourcegraph.accessToken` in VS Code Settings File.
* Updated the docs about instructions to get the access token.

---

## Description

This PR aims to fix the following URL in VS Code Extension Settings:

image

& also, apart from ` ${SOURCEGRAPH_URL}` further URL is incorrect now. `https://sourcegraph.com/users/settings/tokens` redirects to 404: 

image

Correct URL is: `https://sourcegraph.com/users/<sourcegraph-username>/settings/tokens`

## Test plan

- Change current directory to `client/vscode`
- Install Dependencies
- Build & Generate a `.vsix` file
- Install the file and see the difference

### CODE REVIEW

**Comment by Siddhant-K-code** [comment]
> We require contributors to sign our Contributor License Agreement (CLA), and we don't have yours on file. In order for us to review and merge your code, please sign [CLA](https://forms.gle/YnmetmopXNxFxsDUA) to get yourself added.

Submitted the form ✅

File: client/vscode/package.json [hunk 1]
```diff
@@ -116,7 +116,7 @@
             "string"
           ],
           "default": "",
-          "description": "The access token to query the Sourcegraph API. Create a new access token at ${SOURCEGRAPH_URL}/users/settings/tokens"
+          "description": "The access token to query the Sourcegraph API. Create a new access token at https://sourcegraph.com/users/<sourcegraph-username>/settings/tokens"
```
Comment (beyang) [inline client/vscode/package.json:119]:
```suggestion
          "description": "The access token to query the Sourcegraph API. Create a new access token at ${SOURCEGRAPH_URL}/users/<sourcegraph-username>/settings/tokens. Unless you are using a private instance of Sourcegraph, then ${SOURCEGRAPH_URL} is https://sourcegraph.com."
```

File: client/vscode/README.md [hunk 2]
```diff
@@ -99,7 +99,7 @@ This extension contributes the following settings:
...
 | --------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------ |
 | sourcegraph.url                   | Specify your on-premises Sourcegraph instance here, if applicable. The extension is connected to Sourcegraph Cloud by default.                  | "https://your-sourcegraph.com"                               |
-| sourcegraph.accessToken           | The access token to query the Sourcegraph API. Required to use this extension with private instances.                                           | "6dfc880b320dff712d9f6cfcac5cbd13ebfad1d8"                   |
+| sourcegraph.accessToken           | The access token to query the Sourcegraph API. Required to use this extension with private instances. Create a new access token at `https://sourcegraph.com/users/<sourcegraph-username>/settings/tokens`                                           | "6dfc880b320dff712d9f6cfcac5cbd13ebfad1d8"                   |
```
Comment (beyang) [inline client/vscode/README.md:102]:
```suggestion
| sourcegraph.accessToken           | The access token to query the Sourcegraph API. Required to use this extension with private instances. Create a new access token at `${SOURCEGRAPH_URL}/users/<sourcegraph-username>/settings/tokens`                                           | "6dfc880b320dff712d9f6cfcac5cbd13ebfad1d8"                   |
```

File: client/vscode/package.json (hunk 1 above)
Comment (Siddhant-K-code) [reply client/vscode/package.json:119]: Yeah, this looks good. Thanks ✨

**Comment by muratsu** [comment]
Looking into this

**Comment by jdorfman** [comment]
@muratsu any ideas?

**Review by taras-yemets — APPROVED** [review]

**Comment by taras-yemets** [comment]
@Siddhant-K-code, thank you for your PR!
You may need to update the PR (add another commit) so that `verification/cla-signed` pipeline fetches the updated [contributors list](https://github.com/sourcegraph/clabot-config/blob/main/.clabot#L51), which includes your GitHub handle.

**Comment by Siddhant-K-code** [comment]
sent an empty commit, this should work now. Thanks @taras-yemets

**Comment by taras-yemets** [comment]
@Siddhant-K-code, could you please [double check the formatting](https://github.com/sourcegraph/sourcegraph/blob/main/package.json#L14) in `client/vscode/README.md`, and then I'll [run `buildkite/sourcegraph` step again](https://docs.sourcegraph.com/dev/contributing/accepting_contribution#request-a-build-directly)?

**Comment by Siddhant-K-code** [comment]
Formatted the file ✅ 

I think we should include the following command in GitHub action also:

```sh
yarn run format
```

**Comment by philipp-spiess** [comment]
> I think we should include the following command in GitHub action also:

Ideally yes but that would mean a GitHub action needs permission to commit on your behalf and there are a bunch of issues that would come from that so we unfortunately have to do it manually for now. 😞  Thanks for doing that!
